lms/templates/static_templates/embargo.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
```

Comment:
As we discussed earlier, I think it's worthwhile wrapping the call to `_(...)` below with a call to `Text` in case anyone adds HTML to the string in the future.